packages/@sparrow-workspaces/src/features/rest-explorer/components/response-body-navigator/ResponseBodyNavigator.svelte

```diff
@@ -228,6 +238,20 @@
               loader={false}
             />
           </Tooltip>
+        {:else}
+          <Tooltip title={"Export"}>
+            <WithButtonV6
+              icon={DownloadIcon2}
+              onClick={() =>
+                handleDownloadResponse(
+                  formatedBody,
+                  response.contentType,
+                  fileNameWithExtension,
+                )}
```

Comment:
you will get entire headers in **response.headers**, no need to set it additionally in **viewmodel** as **contentType** key

Reply:
it's right but type of response.headers is object[ ] . To access content-type from this have do something as 

A). const contentType = response.headers.find(header => header.key === 'content-Type')?.value;

B). so for the cleaner syntax i have added specific field.

let me know which one to proceed with A or B.

Reply:
for now go with A)
in future we will refactor (we will store headers directly in `object` instead of `array`).

I will create a separate ticket for that.

Reply:
> for now go with A) in future we will refactor (we will store headers directly in `object` instead of `array`).
> 
> I will create a separate ticket for that.

Here it is, review latest commits.